contracts/cw1-whitelist/src/msg.rs

```diff
@@ -46,3 +46,35 @@ pub struct AdminListResponse {
     pub admins: Vec<String>,
     pub mutable: bool,
 }
+
+impl AdminListResponse {
```

Comment:
This is only intended for use in test code, right?

I would not enforce it with `#[cfg(test)]`, but at least add a line in the comment to mention it.

Reply:
I agree, I've considered it. The only reason why I didn't put `#[cfg(test)]` is, that it prevented it from running example as doctest, but it is not a good reason tbh, and I think I am leading to conditional compile it (either as `#[cfg(test}]` or `#[cfg(feature=test-utils)]`.

Reply:
Yes, let's indicate this and the `Allowance` above are for testing purposes.

If these turn out to be useful in multiple places / contracts, we may consider a `test_helpers.rs` somewhere.